lib/diego/bbs/models/desired_lrp_pb.rb

```diff
@@ -15,6 +15,7 @@
 require 'image_layer_pb'
 require 'metric_tags_pb'
 require 'sidecar_pb'
+require 'log_rate_limit_pb'
```

Comment:
This and the other `*_pb.rb` files are generated by this [job](https://ci.cake.capi.land/teams/main/pipelines/capi/jobs/bump-bbs-protos/builds/23) in our pipeline.  They will likely get overridden until a diego release is cut containing your [bbs changes](https://github.com/cloudfoundry/bbs/commit/3a12fde9c836a527f5f32d9edb402c9c36f28a17).  Is that a problem?

Reply:
looking close at our pipeline, it only triggers on diego releases, so it is probably ok.  Unless diego cuts a release without your bbs changes, then it will be a sad time